lib/Fullness/MockDistance.hpp

```diff
@@ -0,0 +1,25 @@
+#include "IDistance.hpp"
+
+/**
+ * @brief Mock distance class that inherits from the IDistance interface class.
```

Comment:
Add file header comments at the top of the file like in this [example in the docs](https://zotbins.github.io/ZotbinsCore/md__github_workspace_docs_doxygen.html#autotoc_md5)